Source/OxyPlot/Series/AreaSeries.cs

```diff
@@ -33,14 +33,25 @@ public class AreaSeries : LineSeries
         /// </summary>
         private List<DataPoint> actualPoints2;
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref = "AreaSeries" /> class.
-        /// </summary>
-        public AreaSeries()
+		/// <summary>
+		/// The index of the data item at the start of visible window
+		/// </summary>
+		private int winIndex;
```

Comment:
Persisting the last visible window index is important as it allows for constant rendering time at any position. When we are rendering a window at the start of the series, we find start index pretty quickly. But when we are closer to the end, it takes more time to find the index. Persisting the last index gives us a starting point for the search, since there is a high chance that next rendering window will be near the last one (zooming and panning mostly occurs gradually with small deltas).